updater/spec/dependabot/updater/operations/empty_return_update_checker.rb

```diff
@@ -0,0 +1,40 @@
+# typed: true
+# frozen_string_literal: true
+
+module Dependabot
+  module EmptyReturn
+    class UpdateChecker < Dependabot::UpdateCheckers::Base
```

Comment:
I don't follow what you mean about a "Bundler situation", could you clarify? I landed on using fake classes because the pattern seems to be to register a class (not an instance of the class) that will then be created to run the required items against. Did you maybe mean to suggest that we use `allow_any_instance_of` to override the behavior of the bundler `update_checker.rb` (and switch our test object to call bundler instead of the made up "emptyreturn"? If I do that in a `before` block does it stay contained to the relevant tests/context? I'm new to Ruby and unsure about how contexts/state is maintained (for example the requires above are no doubt missing because my instinct from C# is "if it compiles it has all the 'usings' it needs.").

Reply:
Sometimes it's easier to write some code than to explain. I think this should work: b078bec5619e0eb9ba7c2d435c7e444c1eafe5d4

> Did you maybe mean to suggest that we use allow_any_instance_of to override the behavior of the bundler update_checker.rb (and switch our test object to call bundler instead of the made up "emptyreturn")?

Yes!

> If I do that in a before block does it stay contained to the relevant tests/context?

Yes!

Reply:
I integrated that commit in manually, (trying to make sure I understand it and I already had some changes from other feedback). Does that look good?